rebar.config

```diff
@@ -50,7 +51,18 @@
                                     {add, cf, [{erl_opts, [debug_info]}]},
                                     {add, cth_readable, [{erl_opts, [debug_info]}]},
                                     {add, eunit_formatters, [{erl_opts, [debug_info]}]}]},
-                       {erl_opts, [debug_info]}]}
+                       {erl_opts, [debug_info]}]},
+
+            {nightly, [
+              {deps, [
+                {relx, {git, "https://github.com/erlware/relx.git",
+                            {branch, "master"}}}
+              ]},
```

Comment:
This will not work. Rebar3 works from the lock file anyway. We shouldn't do a nightly build that does not represent the current repository state IMO anyway. THe nightly build would help people get the bleeding edge master pre-built, but there's no reason to have a nightly on a relx master that we may very well just not ship with at a later point. This creates a counterproductive nightly that won't relate well to the final releases we'll build.